aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -340,6 +344,29 @@ mod loader {
             self
         }
 
+        /// Override the endpoint URL used for **all** AWS services.
+        ///
+        /// This method will override the endpoint URL used for **all** AWS services. This mainly
+        /// exists to set a static endpoint for tools like `LocalStack`. For live traffic, AWS services
+        /// require the service-specific endpoint resolver they provide.
```

Comment:
I think this could be improved a bit so I took a shot at it.
```suggestion
        /// exists to set a static endpoint for tools like `LocalStack`. This can not be used to send requests to
        /// AWS services as they require the service-specific endpoint resolvers that this option overrides.
```